src/pages/store/ProductShowPage.js

```diff
@@ -95,6 +98,19 @@ function ProductShowPage({ match }) {
                     Add to cart
                   </button>
 
+                  <div
+                    style={{ marginTop: "20px" }}
+                    className="has-text-centered"
+                  >
+                    <Display
```

Comment:
you'll need to add an additional prop to this component:
`config={{ type: 'pdp' }}`
otherwise it will display the payment method and not the PDP banner.

Reply:
Will do. Thanks.

Reply:
Updated.